### PR TITLE
.Net: added readme

### PR DESCRIPTION
**Pull request by yorek**
Added readme and sample for Microsoft.SemanticKernel.Connectors.SqlServer

### CODE REVIEW

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+Add the Semantic Kernel packages needed to create a Chatbot:
+
+```bash
+dotnet add package Microsoft.SemanticKernel
+dotnet add package Microsoft.SemanticKernel.Connectors.OpenAI
+```
+
+Add then add `Microsoft.SemanticKernel.Connectors.SqlServer` to give your Chatbot memories:
```
Comment (dmytrostruk) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:23]:
```suggestion
Add `Microsoft.SemanticKernel.Connectors.SqlServer` to give your Chatbot memories:
```

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+```bash
+dotnet add package Microsoft.SemanticKernel.Connectors.SqlServer
```
Comment (dmytrostruk) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:26]:
```suggestion
dotnet add package Microsoft.SemanticKernel.Connectors.SqlServer --prerelease
```

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+#pragma warning disable SKEXP0001, SKEXP0010, SKEXP0020
+
+// Replace with your Azure OpenAI endpoint
+const string AzureOpenAIEndpoint = "https://.openai.azure.com/";
+
+// Replace with your Azure OpenAI API key
+const string AzureOpenAIApiKey = "";
+
+// Replace with your Azure OpenAI deployment name
```
Comment (dmytrostruk) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:47]:
```suggestion
// Replace with your Azure OpenAI embedding deployment name
```

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+const string EmbeddingModelDeploymentName = "embeddings";
+
+// Replace with your Azure OpenAI deployment name
```
Comment (dmytrostruk) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:50]:
```suggestion
// Replace with your Azure OpenAI chat completion deployment name
```

**Review by dmytrostruk — APPROVED** [review]
Thanks for adding documentation!

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    Console.Write("Question: ");
+    var question = Console.ReadLine()!;
+
+    Console.WriteLine("\nSearching information from the memory...");
+    builder.Clear();
+    await foreach (var result in memory.SearchAsync(TableName, question, limit: 3))
+    {
+        builder.AppendLine(result.Metadata.Text);
+    }
+    if (builder.Length != 0) {
```
Comment (cincuranet) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:88]: nit: The `{` should be on next line (consistent with rest of the file).

**Review by cincuranet — APPROVED** [review]

**Comment by cincuranet** [comment]
LGTM

File: dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md (hunk 5 above)
Comment (markwallace-microsoft) [inline dotnet/src/Connectors/Connectors.Memory.SqlServer/README.md:88]:
```suggestion
    if (builder.Length != 0)
    {
```

**Review by markwallace-microsoft — APPROVED** [review]

**Review by westey-m — APPROVED** [review]

**Comment by yorek** [comment]
Thanks!!!

**Pull request merged**